packages/react/README.md

```diff
@@ -0,0 +1,72 @@
+<div align="center">
+  <p>
+    <img alt="Gadget logo" src="https://raw.githubusercontent.com/gadget-inc/js-clients/main/docs/assets/gadget-logo.png" />
```

Comment:
This won't work until we merge. I didn't want to do a relative path (`../../`), but maybe that'd be okay?